src/tpm12/tpm_nvram.c

```diff
@@ -1290,7 +1290,7 @@ TPM_RESULT TPM_Process_NVReadValue(tpm_state_t *tpm_state,
     TPM_BOOL			physicalPresence;
     TPM_BOOL			isGPIO = FALSE;
     BYTE 			*gpioData = NULL;
-    TPM_NV_DATA_SENSITIVE	*d1NvdataSensitive;
+    TPM_NV_DATA_SENSITIVE	*d1NvdataSensitive = NULL;
```

Comment:
From looking at `TPM_NVIndexEntries_GetEntry()`, which would return an error code in case it could not initialize d1NvdataSensitive (found = FALSE) and otherwise have it initialized, this seems to be a false positive.

Reply:
A false positive nowadays, but only if analyzed down to the level you did and only as long as `TPM_NVIndexEntries_GetEntry` or other potential future users are not changing the way they work.
Is there any drawback in initializing them explicitly?

Reply:
> Is there any drawback in initializing them explicitly?

I don't mind these patches but I also have to determine whether they are false positives or real issues because in the end a NULL pointer dereference isn't much better.

Reply:
yeah, thanks for sharing your thoughts